nextflow_schema.json

```diff
@@ -68,122 +43,247 @@
-                    "default": 1e-10
+                    "default": 1e-10,
+                    "description": "Evalue cutoff",
+                    "help_text": "Sets the cutoff value for Evalue\n\n> Modifies DeepARG parameter: `--arg-alignment-evalue`",
+                    "fa_icon": "fas fa-align-center"
                 },
                 "arg_deeparg_alignmentidentity": {
                     "type": "integer",
-                    "default": 50
+                    "default": 50,
+                    "description": "Identity cutoff for sequence alignment",
+                    "help_text": "Sets the value for Identity cutoff for sequence alignment\n\n> Modifies DeepARG parameter: `----arg-alignment-identity`",
+                    "fa_icon": "fas fa-align-center"
                 },
                 "arg_deeparg_alignmentoverlap": {
                     "type": "number",
-                    "default": 0.8
+                    "default": 0.8,
+                    "description": "Alignment read overlap",
+                    "help_text": "Sets the value for the allowed alignment read overlap.\n\n> Modifies DeepARG parameter: `--arg-alignment-overlap`",
+                    "fa_icon": "fas fa-align-center"
                 },
                 "arg_deeparg_numalignmentsperentry": {
                     "type": "integer",
-                    "default": 1000
-                }
-            }
-        },
-        "arg_rgi": {
-            "title": "ARG-RGI",
-            "type": "object",
-            "description": "Parameters for RGI - Resistance Gene Identifier",
-            "default": "",
-            "properties": {
-                "arg_skip_rgi": {
-                    "type": "boolean"
+                    "default": 1000,
+                    "description": "Diamond, minimum number of alignments per entry",
+                    "help_text": "Sets the value of minimum number of alignments per entry for Diamond.\n\n> Modifies DeepARG parameter: `--arg-num-alignments-per-entry`",
+                    "fa_icon": "far fa-gem"
                 }
-            }
+            },
+            "fa_icon": "fas fa-tools"
         },
-        "amr_fargene": {
-            "title": "AMR-fargene",
+        "arg_fargene": {
+            "title": "ARG-fARGene",
             "type": "object",
-            "description": "Parameters for fargene",
+            "description": "Antimicrobial resistance gene detection, using a deep learning model",
             "default": "",
             "help_text": "[fARGene](https://github.com/fannyhb/fargene) (Fragmented Antibiotic Resistance Gene Identifier ) is a tool that takes either fragmented metagenomic data or longer sequences as input and predicts and delivers full-length antiobiotic resistance genes as output. The tool includes developed and optimized models for a number or resistance gene types, and the functionality to create and optimize models of your own choice of resistance genes.",
```

Comment:
If you want to do this, we should add a description for each tool - I think this would actually be useful, and you can just copy and past from your output docs already.

Howeve ri tseems markdown doesn't work for teh group description box, so we should remove the link (or the square brakects)

![image](https://user-images.githubusercontent.com/17950287/178688908-5e4d66c6-b0bc-402e-aa73-7ed663b1bd78.png)

Reply:
I've done this for everything in https://github.com/nf-core/funcscan/pull/78/commits/a0a69fe617000f34171afa865adda2de87c398a9